posts/2023-09-29-open-liberty-on-aws-ecs-fargate.adoc

```diff
@@ -0,0 +1,684 @@
+---
+layout: post
+title: "Running Open Liberty on Amazon Elastic Container Service with AWS Fargate"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/abutch3r
+author_github: https://github.com/abutch3r
+seo-title: Running Open Liberty on Amazon Elastic Container Service with AWS Fargate - OpenLiberty.io
+seo-description: How to run Open Liberty in Amazon Elastic Container Service with AWS Fargate
+blog_description: "How to run Open Liberty in Amazon Elastic Container Service with AWS Fargate"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Running Open Liberty on Amazon Elastic Container Service with AWS Fargate
+Alex Butcher <https://github.com/abutch3r>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+Serverless applications continue to be a hot topic in 2023. With MicroProfile being widely adopted by cloud native applications, many developers want to know:
```

Comment:
Maybe change this to be something like: "Serverless application continue to rise in popularity as developers seek rapid deployments, reduced operational load and lower costs. With MicroProfile....."
At the moment, this doesn't feel very persuasive or give much of an idea as to why it's popular or a trend. Would be good to include this.